src/codewhisperer/util/runtimeLanguageContext.ts

```diff
@@ -39,6 +46,25 @@ export class RuntimeLanguageContext {
             shellscript: 'shell',
             sql: 'sql',
         })
+        this.supportedLanguageExtensionMap = createConstantMap<CodeWhispererConstants.SupportedLanguage, string>({
```

Comment:
some languages like yaml have extension aliases (.yaml, .yml) so this might need to be `string[]`

Reply:
This is actually fine, what we are doing here to try to map the language type to any appropriate extension to be used by the service to generate completion for that language, so in yaml case, we can choose either .yaml or .yml the service would know to generate code for yaml(yaml here just an example since we don't support yaml yet)